_components/mqss_mqp_dashboard_frontend

```diff
@@ -0,0 +1,14 @@
+---
+title: MQSS MQP Dashboard Frontend
+languages:
+  - JavaScript
+frameworks:
+  - React
+links:
+  docs:
+  github: https://github.com/Munich-Quantum-Software-Stack/MQP-Dashboard-Frontend
+maintainers:
+  - LRZ (QCT)
+---
+
+Web bashboard to view available back-ends, generate personal toket, display run jobs with details. Provide also access to telemetry data.
```

Comment:
```suggestion
Web dashboard to view available backends, generate a personal access token, and display detailed job results. Also provides access to telemetry data.
```
@mdepasca-lrz Would be good to request a review next time. 
There were several typos in this submission.
Also, the `docs` link is empty and I am pretty sure that breaks the webpage.